src/text/text_writer.rs

```diff
@@ -26,6 +26,22 @@ impl TextWriterBuilder {
         }
     }
 
+    /// Constructs a newline-delimited text Ion writer that adds UNIX and human-friendly newlines
+    /// between top-level values.
+    ///
+    /// For example:
+    /// ```ignore
+    /// {foo: 1, bar: 2, baz: 3}
+    /// [1, 2, 3]
+    /// true
+    /// "hello"
+    /// ```
+    pub fn lines() -> TextWriterBuilder {
+        TextWriterBuilder {
+            text_kind: TextKind::Newline,
```

Comment:
Could we change `TextKind::Newline` to `TextKind::Lines` for consistency?

Reply:
Done!